modules/metering-reports.adoc

```diff
@@ -4,15 +4,16 @@
 [id="metering-reports_{context}"]
 = Reports
 
-The `Report` custom resource is used to manage the execution and status of reports.
+The Report custom resource is used to manage the execution and status of reports.
 Metering produces reports derived from usage data sources, which can be used in further analysis and filtering.
 
-A single `Report` resource represents a job manages a database table and updates it with new information according to a schedule, and exposes the data in that table via the reporting-operator HTTP API.
-Reports with a `spec.schedule` field set are always running, and will track what time periods it has collected data for, ensuring that if Metering is shutdown or unavailable for an extended period of time, it will backfill the data starting where it left off.
+A single Report resource represents a job manages a database table and updates it with new information according to a schedule, and exposes the data in that table via the reporting-operator HTTP API.
+Reports with a `spec.schedule` field set are always running, and will track what time periods it has collected data for, ensuring that if metering is shutdown or unavailable for an extended period of time, it will backfill the data starting where it left off.
 If the schedule is unset, then the Report will run once for the time specified by the `reportingStart` and `reportingEnd`.
-By default, reports will wait for `ReportDataSources` to have the progressed in their import process to cover the report period being processed.
+By default, reports will wait for ReportDataSources to have the progressed in their import process to cover the report period being processed.
```

Comment:
Please clarify this sentence.

Reply:
Reports wait for the necessary data to import before running. I updated these two sentences. Good call.